main.ps1

```diff
@@ -14,7 +14,7 @@
                           -CMAKE_BUILD_CONFIGURATION: Debug `
                           -FOMOD_INCLUDE_PDB $true `
                           -FOMOD_REQUIRED_INSTALLATION_DIR 'FOMOD/Required Files' `
-                          -FOMOD_MOD_NAME 'NPCs Names Distributor Debug' `
```

Comment:
oops, busted 😄 to be honest .ps1 file is redundant and is more for local debugging of CI script.

Reply:
Yeah I just changed it when I saw it locally :P Was kinda nice to use it to test the fomod locally